test/stability/testCksum/checksum_test.go

```diff
@@ -8,9 +8,36 @@ import (
 	"testing"
 )
 
+
+func TestInit(t *testing.T) {
+	if err := initTest(); err != nil {
+        t.Fatalf("fail to initialize with error: %+v\n", err)
+    }
+}
+// TODO unfortunately this test can't be executed as go test right now due to https://github.com/intel-go/nff-go/issues/301
+// If you want to test it via go test you should firstly comment FreeMempools function at flow/flow.go
+// You will have multiple mempools however test will pass.
 func TestChecksum(t *testing.T) {
-	if err := executeTest(0); err != nil {
-		t.Logf("fail: %+v\n", err)
-		t.Fail()
+	variants := []bool {true, false}
```

Comment:
Too bizzare. I would go with test pattern table variants[][] bool {{false, false,false,false,false,false},} This will eliminate nested fors and check for presence of ICMP only with tcp/udp.